server/handlers/events_streamer.go

```diff
@@ -134,9 +134,36 @@ STOP:
 		}
 		time.Sleep(5 * time.Second)
 	}
+	close(respChan)
 	defer log.Debug("events handler closed")
 }
+func listenForCoreEvents(ctx context.Context, eb *events.EventStreamer, resp chan []byte, log *logrus.Entry, p models.Provider) {
+	log.Debugf("listening for core events")
```

Comment:
Can probably remove now. It might be nice not to see this while running locally.